### PR TITLE
#6 Add support of credit operations

### PR DESCRIPTION
**Pull request by ihorml**
**Issue**: #6 

**Description**:

Screenshot 2022-10-11 at 14 06 30

Screenshot 2022-10-11 at 14 06 42

> @sschiessl-bcp 
> Do credit offer first, then give me an estimate what you think you ll need for all

Answering this - I suppose it's up to 1-2 hours to integrate all the operations from refUI (I found the code with all translations/formats/colors, so technically it's just copy-paste and possibly may take even less time, but not sure how to properly test it, maybe just hardcode the `operation`)

### CODE REVIEW

File: app/core/app.utilities.js [hunk 1]
```diff
@@ -1224,6 +1240,110 @@
...
+                }
+                else if (operation_type === 73) { // Credit Offer Accept
+                    const operation_account = operation.account;
+                    const repay = operation.repay_amount;
+                    const fee = operation.credit_fee;
+                    getAccount(operation_account).then((account_name) => {
+                        getAsset(repay.asset_id, repay.amount).then((repayAsset) => {
+                            getAsset(fee.asset_id, fee.amount).then((feeAsset) => {
+                                operation_text = $filter('translateWithLinks')('Operation Credit Deal Repay', {
+                                    accountLink : {
```
Comment (sschiessl-bcp) [inline app/core/app.utilities.js:1326]: Since you introduce shortcut functions, maybe also do that for links?

**Review by sschiessl-bcp — CHANGES_REQUESTED** [review]
Ok, please add the rest

**Comment by ihorml** [comment]
@sschiessl-bcp pushed

File: app/core/app.utilities.js (hunk 1 above)
Comment (ihorml) [reply app/core/app.utilities.js]: true, pushed

**Pull request merged**